README.md

```diff
@@ -91,6 +91,10 @@ Short description.
 
   Offers a range of products based on climate and environmental research data, where you can monitor your personal footprint in their climate calculator, learn about climate psychology, and reduce carbon emissions through friendly team competitions.
 
+- [Unmade](https://unmade.com) | England | 2013
+
+  Unmade is a global fashion software company driving innovation through customisation and delivering an end-to-end digital solution for on-demand production at scale. Unmade was born out of a desire to reimagine both the design and manufacturing processes within the fashion industry, so brands only make what is actually sold, resulting in minimal waste.
```

Comment:
Hey, thank you for your contribution!

Yup, the category is correct. I will actually rename it to `Social Impact`, as `Awareness` doesn't make its entries justice. Those companies are not just about making people know about problems, but they're also about taking action. 

One thing though, I would update the description to not use buzz words (eg. driving innovation, digital solution) and just explain simply what they do. Here's my suggestion:

```suggestion
Reduces waste in the fashion industry, by offering on-demand production of clothing at scale. Unmade was born out of a desire to re-imagine both the design and manufacturing processes within the fashion industry, so brands only make what is actually sold, resulting in minimal waste.
```